### PR TITLE
Release/v0.10.0

### PR DESCRIPTION
**Pull request by maxvisser**
changelog:
- New provider signupflow
- settings page: email settings
- Unassigned vouchers, add pdf export option
- Modal fund request clarification, fix icon overflow
- Webshop, update some page titles
- Modal pdf preview, update translation
- Make CSV upload on voucher generator qdt-env configurable via flags
- Dashboards, add organization switcher
- Allow signups usage on tablets
- Berkelland: Provider landing page, fix header image issue on IE
- Fund selector adjustments
- Navbar, replace logo with higher quality
- refactoring, feedback, removed alerts and redirects on some pages
- New Forus Website
- Fund invitation accept, solve form lock issue
- add target parameter to auth-link router for Me app voucher info

### CODE REVIEW

**Review by GerbenBosschieter — APPROVED** [review]
- Webshop frontend bug with IIE. Only displays footer. Should be fixed before release

**Review by GerbenBosschieter — CHANGES_REQUESTED** [review]
Accidentally approved last review.
- Webshops do not work on IE.
- Website doesn't work on IE (this was already the case, but would be nice to also fix this for the release)
- Sign up v2 doesn't work either on Ie

**Comment by maxvisser** [comment]
when SESSIONS_GEO_IP_ENABLED is set to true Internet explorer will stop working. (on backend); issue is different from the one preventing signup-v2 and website not working

**Comment by maxvisser** [comment]
Still should be changed: hide email settings with a qdt-env flag from frontend

**Review by GerbenBosschieter — CHANGES_REQUESTED** [review]
Can't acces pages on IE:
- Provider dashboard
- Signup flows v1
- Also mentioning https://test-inkomensondersteuning.nijmegen.nl/provider/#!/ . Not sure but this but I think this is a separate pug file.

File: src/sources/forus-platform/pug/layout/includes/scripts.pug [hunk 1]
```diff
@@ -1,7 +1,13 @@
 - var src_prefix = (qdt_c.platform.env_data.html5ModeEnabled?qdt_c.platform.env_data.html5Mode.basePath:'');
 
+//- google maps
```
Comment (maxvisser) [inline src/sources/forus-platform/pug/layout/includes/scripts.pug]: should be loaded after app.js and bundle.js

**Review by GerbenBosschieter — APPROVED** [review]

**Pull request merged**